src/Manifest/Manifest.router.ts

```diff
@@ -68,6 +73,14 @@ export class ManifestRouter extends Router {
     }
   }
 
+  async getPublicProjectManifest(req: AuthRequest) {
```

Comment:
Is this method necessary? can't we just use `getProjectManifest`? (the `public` aspect of this seems to be the lack of middlewares)